prs/ChicagoStyleSausagePizza.ts

```diff
@@ -0,0 +1,13 @@
+class ChicagoStyleSausagePizza extends Pizza {
+    public ChicagoStyleSausagePizza() {
```

Comment:
This is not the correct syntax for `constructors` in typescript.  Take a look at the [docs](https://www.typescriptlang.org/docs/handbook/2/classes.html#constructors) for more info.